esp/src/eclwatch/WUQueryWidget.js

```diff
@@ -456,6 +457,13 @@ define([
             this.workunitsGrid.onSelectionChanged(function (event) {
                 context.refreshActionState();
             });
+
+            aspect.after(this.workunitsGrid, 'gotoPage', function (promise, args) {
```

Comment:
(slight correction to my example)
```
aspect.after(this.workunitsGrid, 'gotoPage', function(deferred){
  return deferred.then(function(pageNum){
    pageNum > 1 ? context._idleWatcher.stop() : context._idleWatcher.start();
  });
});
```
I am sure both work - in their docs they mirror the actual params in the function callback (see numPages above), where your extracting the first param from the args array.  (My question was just a sanity check).